jsapp/js/account/plans/confirmChangeModal.component.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import cx from 'classnames';
+
```

Comment:
`cx` is actually a regular export from classnames, for convenience (you can see it used on the repo's [how-to](https://github.com/JedWatson/classnames).) The `cx` name is a [bit of an artifact](https://github.com/JedWatson/classnames/issues/157), but that's an upstream decision. You're right that it could be confusing; I admit I got tripped up by it when I initially learned by it. Do you think a comment above the import would be acceptable?

Reply:
For context, the reason I prefer using `cx` over `classnames` is that `className={classname(style.foo, style.bar)}` tends to be harder (for me, at least) to read than `className={cx(style.foo, style.bar)}`, especially when we're feeding in more complex parameters.

Reply:
I buy the explanation. Let's leave `cx` :) I agree that `className={classNames…` is hard to read